src/addRemove.js

```diff
@@ -0,0 +1,141 @@
+/* eslint-disable no-plusplus */
+import './style.css';
+import dragDrop from './dragdrop.js';
+import statusUpdate from './server.js';
+
+class Task {
+  constructor() {
+    this.size = 0;
+    this.ul = document.getElementById('task-list');
+    this.savedTasks = JSON.parse(localStorage.getItem('savedTasks')) || [];
+  }
+
+  clearContent() {
+    this.savedTasks = this.savedTasks.filter((object) => object.completed !== true);
+    localStorage.setItem('savedTasks', JSON.stringify(this.savedTasks));
+    this.displayAllTask();
+  }
+
+  addIndex() {
+    if (this.savedTasks.length < 1) {
+      return this.size;
+    }
+    return this.savedTasks[this.savedTasks.length - 1].index + 1;
+  }
+
+  addTask(book) {
+    this.savedTasks.push(book);
+    localStorage.setItem('savedTasks', JSON.stringify(this.savedTasks));
+    this.size += 1;
+  }
```

Comment:
- [OPTIONAL] I suspect something might not be semantically right. Is it a task or book that we are adding? Consider looking into it to add clarity to this part of the code.